src/bindings/python/tests/test_runtime/test_properties.py

```diff
@@ -311,7 +311,7 @@ def test_properties_ro(ov_property_ro, expected_value):
             hints.enable_hyper_threading,
             "ENABLE_HYPER_THREADING",
             (
-                (True, True),
+                (False, True),
```

Comment:
why did you change the value here?

Reply:
It is just an experiment. Have reverted it.